tests/testthat/test-SQLServer.R

```diff
@@ -173,6 +173,26 @@ test_that("SQLServer", {
     expect_equal(as.double(values[[6]]), as.double(received[[6]]))
   })
 
+  local({
+    con <- DBItest:::connect(DBItest:::get_default_context())
+    input <- DBI::SQL(c(
```

Comment:
Can you indent long arguments with a single tab? (https://style.tidyverse.org/syntax.html#long-lines). You might want to use https://github.com/lionel-/codegrip to do it for you.

Reply:
Thanks - tried to address ( manually, will look into using codegrip ).